tests/qam-minimal/s390tools.pm

```diff
@@ -35,7 +36,10 @@ sub run {
     validate_script_output 'lsdasd', sub { m/Bus-ID|Status|Name|Device|Type|BlkSz|Size|Blocks/ };
     validate_script_output 'lsmem', sub { m/RANGE|SIZE|STATE|REMOVABLE|BLOCK/i };
     # dasd_reload does exit with 4
-    assert_script_run 'dasd_reload|grep -E "offline|Activating"';
+    if (script_run('dasd_reload|grep "line 79: echo: write error"') == 0) {
+        record_soft_failure 'bsc#';
+        script_run 'dasd_reload|grep -E "offline|Activating"';
+    }
```

Comment:
This should also go to another PR and the BSC number is missing.

Reply:
I know :smile: I just wanted to run it in serial terminal to copy the output. WIP